config.go

```diff
@@ -332,6 +332,13 @@ func loadConfig() (*config, error) {
 	// path to all of the files and directories that will live within it.
 	lndDir := cleanAndExpandPath(preCfg.LndDir)
 	if lndDir != defaultLndDir {
+		preCfg2 := config{} // Parse again the command line to check if a config file is set
+		if _, err := flags.Parse(&preCfg2); err != nil {
```

Comment:
There's no need to re-parse the command line flags since we already have the config file path from the first time we parsed them.